src/Components/TodoContainer.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-param-reassign */
```

Comment:
As I said earlier please it would be best to solve any linter errors in your code rather than disabling linter checks. it helps you write better code.